src/x/xHilbertTransform.ts

```diff
@@ -1,26 +1,66 @@
+import { DoubleArray } from 'cheminfo-types';
+import FFT from 'fft.js';
+
+import { xCheck } from './xCheck';
+
 /**
  * Performs the Hilbert transform
  * @returns A new vector with 90 degree shift regarding the phase of the original function
```

Comment:
We should take the habit to add reference if we can: 

 * @link https://en.wikipedia.org/wiki/Hilbert_transform